Contributing.md

```diff
@@ -14,40 +14,73 @@ Conduct](CODE_OF_CONDUCT.md).
 Rustfmt requires a nightly compiler. Replace all invocations of `cargo` by
 `cargo +nightly` if necessary.
 
-At all times the environment variables `CFG_RELEASE_CHANNEL` and `CFG_RELEASE` must
-be set to either `nightly` or `beta`. Both should be set to the same value. The
-project supports [cargo-make](https://github.com/sagiegurari/cargo-make) which
-automatically sets these variables for you:
+At all times the environment variables `CFG_RELEASE_CHANNEL` and `CFG_RELEASE` must be set. `CFG_RELEASE_CHANNEL` must be set to either `nightly` or `beta`, and `CFG_RELEASE` should be set to your corresponding Rust version. The
+project supports and recommends [cargo-make][cargo-make] when working on rustfmt, because `cargo-make` automatically sets these variables for you
 
+The available tasks are listed in `Makefile.toml`.
+
+You can alternatively use `cargo` directly, but you'll have to set the `CFG_RELEASE` and `CFG_RELEASE_CHANNEL` environment variables and also provide the corresponding features.
+
+For example:
+```sh
+export CFG_RELEASE=1.45.0-nightly
+export CFG_RELEASE=nightly
 ```
```

Comment:
Nit: this feels redundant, maybe it's better to put the code example under line 17

Reply:
Thanks for the feedback! Could you elaborate on how/why you feel this example is redundant?

It's the first, and only, occurrence within this file. Arguably folks should know how to do this, but we want to make it as easy as possible for users to contribute to rustfmt and IMHO over communication is better than the alternative in a dev/new-contributor context like this. 

TBH I'm not keen to move this up to line 17 as you suggest, because users don't need to be bothered with this if they use `cargo make` which was the main driver for incorporating that into the project and encouraging it's use. I suppose that the

> `CFG_RELEASE_CHANNEL` must be set to either `nightly` or `beta`, and `CFG_RELEASE` should be set to your corresponding Rust version

line from above could be moved down here though

Reply:
Sorry; this was poorly worded. I mean to say that the text above the code example for exporting the env variables feels redundant, and it may be worth putting the code example for setting the env variables directly underneath where there are mentioned in text before moving on to `cargo make`. This is a very much a nit though :)